libpod/container_api.go

```diff
@@ -262,6 +262,9 @@ func (c *Container) Init() (err error) {
 			return err
 		}
 	}
+	if err := c.setupOCIHooks(&g); err != nil {
```

Comment:
I think this snuck in from another of your PRs

Reply:
oops